controllers/disruption_controller.go

```diff
@@ -881,6 +885,19 @@ func (r *DisruptionReconciler) recordEventOnDisruption(instance *chaosv1beta1.Di
 	}
 }
 
+// sendWarnings is used to send users warnings about their disruption spec that we're unable to detect or warn about in the validation webhook
+func (r *DisruptionReconciler) sendWarnings(instance *chaosv1beta1.Disruption) error {
```

Comment:
```suggestion
func (r *DisruptionReconciler) sendWarningsOnDisruptionSpec(instance *chaosv1beta1.Disruption) error {
```

Reply:
I wanted to leave the more generic title in case we had other uses for sending warnings, but you're right, that's the wrong instinct, we can always rename it later

Reply:
it's fine if we do leave it generic but in that case let's also make the comment generic.